src/core-asyncpipeline/AsyncPipeline/AsyncResultFlow/Equality/Equality.Equals.Wrap.cs

```diff
@@ -0,0 +1,9 @@
+#nullable enable
+
+namespace System
+{
+    partial struct AsyncResultFlow<TSuccess, TFailure>
+    {
+        public override bool Equals(object? obj) => obj is AsyncResultFlow<TSuccess, TFailure> other && Equals(other);
```

Comment:
Prepare for .NET 6:
Change to `object? obj` to `[NotNullWhen(true)] object? obj`

Reply:
Done